### DISCUSSION TITLE
Poll results on z/OS Access to GitHub

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#172] [Reference]
This poll ran for 1 week on Twitter, Facebook mainframe group, IBM-Main Listserv, and other places. It is not a complete, or scientific, poll but it does show what may be a concern for the distribution of ZOT.

![image](https://user-images.githubusercontent.com/42328411/218136302-dc1fe7c8-24ea-4671-a220-3f23ecd8b193.png)

**Comment by IgorTodorovskiIBM** [comment]
I'm curious of the percentage that don't have access to Github, are they not allowed to access any open source period or is just github.com blocked? What if the open source tools was hosted on a site like ibm.com?

**Reply by v1gnesh** [reply]
I find it surprising that about half the responders have GitHub access from mainframe.
Did they mean GitHub SaaS or GitHub Enterprise?

The friction is more to do with customer environments, as an operational thing.
When I say "operational", I mean the difference between being agile (without capital A) and being formal to death.
Reviewing information, onboarding things... a process which will easily take MONTHS in any big co.

Eventually, the releases from GitHub need to end up as zOS Portal Software Instances. 
That way, even if they can't get the bleeding edge, PSIs hosted on an IBM site may be acceptable.

**EDIT:** I wonder if you feel gutted (LOL) with just the mention of OSS & zOSMF/ShopZ in the same breath.
That's exactly the friction that I'm talking about.

**Reply by v1gnesh** [reply]
Things to address (ideally):

- transparently taking care of codepage hassle (but with docs to see exactly what's happening)
- install/update stuff from vendors' GitHub/GitLab (ex: IBM-Z-zOS, BroadcomMFD, etc.) -- package mgmt for git-sourced stuff
- SupportPacs, [tapetool](https://public.dhe.ibm.com/storage/tapetool/), things like it from TechDocs (please make TechDocs usable again)
- zOSMF Portable  Software Instances
- allow passing all of the above through Nexus Repository or whatever equivalents are common in enterprises
Not just Nexus Repository SaaS, but on-prem based NR, and IBM/vendors becoming a source that NR can fetch from.

**Comment by lbdyck** [comment]
I would recommend proceeding for github only access for now and after the packaging for that works well, then proceed to other options.

**Comment by wizardofzos** [comment]
WIth 43 responses that's a bit hard to make any conclusions.... also not knowing if it's me responding (based on my ZPDT) or someone from, let's say "VISA" makes the difference....
